<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportLauncher</class>
 <widget class="QDialog" name="ReportLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Projects/Blink/icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxJust">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>147</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string> Apenas</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxPeriod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>52</y>
     <width>380</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string> Periodo</string>
   </property>
   <widget class="QDateTimeEdit" name="dateTimeEditSince">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>31</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditUntil">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>31</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>35</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>At√©</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>36</y>
      <width>20</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>De</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
